README.md

```diff
@@ -117,15 +117,9 @@ To find what FHIR Server supports, use the `GET Metadata` Postman request to ret
 
 **Authorizing a user**
 
-FHIR Works on AWS uses Role-Based Access Control (RBAC) to determine what operations and what resource types a user can access. The default rule set can be found in [RBACRules.ts](https://github.com/awslabs/fhir-works-on-aws-deployment/blob/mainline/src/RBACRules.ts). To access the API, you must use the OAuth access token. This access token must include scopes of either `openid`, `profile` or `aws.cognito.signin.user.admin`. 
```

Comment:
To get groups to be put into the JWT the requester must use these scopes during the request (this was required about a year ago perhaps it changed).

So the idea behind these is the fail early principal (fail at authorized level and not even invoke proxy lambda). That said if removing simplifies the flow I am okay removing it.

Reply:
Looking at the RBAC package if groups is null in JWT token it will be an empty array and will result in "Unauthorized" exception